src/directions/main.ts

```diff
@@ -147,7 +151,7 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
         if (response.code !== "Ok") throw new Error(response.message ?? "An unexpected error occurred.");
       } finally {
         // see #189 (https://github.com/maplibre/maplibre-gl-directions/issues/189)
-        if (this.abortController.signal.reason !== "DESTROY") this.interactive = prevInteractive;
+        if (this.abortController?.signal.reason !== "DESTROY") this.interactive = prevInteractive;
```

Comment:
Another fix, should've been a separate PR, but I decided to include it here not to waste the time.